.trivyignore

```diff
@@ -3,6 +3,7 @@
 
 # =========== CRITICAL ================
 CVE-2024-24790
+CVE-2024-5535
```

Comment:
Thanks Soham, do you know which package has this CVE?

Reply:
Thanks Grace! It is from cbl-mariner package and from openssl library.